public_website/templates/public_website/about.html

```diff
@@ -13,61 +13,26 @@ <h2>{% trans '...and Sawaliram was born!' %}</h2>
     <img src="{% static 'illustrations/about.png' %}" alt="{% trans 'about' %}" align="right">
     
     <p> 
-        {% blocktrans %}
-            It was 1972 and a group of scientists, university academics, graduate students and school teachers were
-            gathering in the village of Rasulia, in the Hoshangabad District of Madhya Pradesh State. 
-            They wished to develop an active, inquiring and experiment-based science program, a curriculum that 
-            was rooted in rural India.Their efforts led to the landmark 'Hoshangabad Science Teaching 
-            Program <a href="https://www.eklavya.in/past-work-top/programmes-past-top/hstp">(HSTP)</a>', one that was implemented in the next 30 years over large tracts of the State.' 
-         {% endblocktrans %}
+    {% trans "It was 1972 and a group of scientists, university academics, graduate students and school teachers were gathering in the village of Rasulia, in the Hoshangabad District of Madhya Pradesh State. They wished to develop an active, inquiring and experiment-based science program, a curriculum that was rooted in rural India.Their efforts led to the landmark 'Hoshangabad Science Teaching Program" %}<a href="https://www.eklavya.in/past-work-top/programmes-past-top/hstp">(HSTP)</a>'{%trans ", one that was implemented in the next 30 years over large tracts of the State." %}
```

Comment:
can you please try writing the paragraph writing in multiple lines as in before

Reply:
How about now ?